packages/components/pie-toast/test/component/pie-toast.spec.ts

```diff
@@ -17,4 +35,162 @@ test.describe('PieToast - Component tests', () => {
         // Assert
         expect(toast).toBeVisible();
     });
+
+    test.describe('Props', () => {
+        test.describe('isOpen', () => {
+            test('should not render component if isOpen is false', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieToast, {
+                    props: {
+                        isOpen: false,
+                    } as ToastProps,
+                });
+
+                // Act
+                const toast = page.locator(componentSelector);
+
+                // Assert
+                expect(toast).not.toBeVisible();
+            });
+        });
+
+        test.describe('message', () => {
+            test('should render the message', async ({ mount, page }) => {
+                // Arrange
+                const confirmMessage = 'Item has been created';
+                await mount(PieToast, {
+                    props: {
+                        message: confirmMessage,
+                    } as ToastProps,
+                });
+
+                // Act
+                const toast = page.locator(componentSelector);
+                const message = page.locator(messageSelector);
+
+                // Assert
+                expect(toast).toBeVisible();
+                expect(message).toBeVisible();
+                expect(message).toHaveText(confirmMessage);
+            });
+
+            test('should not render the message if is empty string', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieToast, {
+                    props: {
+                        message: '',
+                    } as ToastProps,
+                });
+
+                // Act
+                const toast = page.locator(componentSelector);
+                const message = page.locator(messageSelector);
+
+                // Assert
+                expect(toast).toBeVisible();
+                expect(message).not.toBeVisible();
+            });
+        });
+
+        test.describe('isDismissible', () => {
+            test('should not show the close icon if isDismissible is false', async ({ mount, page }) => {
```

Comment:
May need another test for when it is `true`.

Reply:
Its already true by default and the very first test case already test if the component is visible. The reason that I am omitting this is last time when I build `Notification` It was told me to remove tests that has default props.